pkg/api/v1/types.go

```diff
@@ -2165,6 +2165,14 @@ type PodProxyOptions struct {
 	Path string `json:"path,omitempty"`
 }
 
+// ServiceProxyOptions is the query options to a Service's proxy call.
+type ServiceProxyOptions struct {
+	unversioned.TypeMeta `json:",inline"`
+
+	// Path is the URL path to use for the current proxy request to service.
```

Comment:
Comment isn't great, but you never interact with this object directly... it gets populated from the path during API routing